package.json

```diff
@@ -26,10 +26,12 @@
     "build": "electron-vite build",
     "dist": "electron-builder",
     "coverage": "sensible-browser coverage/index.html",
+    "desktop": "electron-vite dev",
     "format": "prettier --write \"client/**/*.ts*\" && eslint --fix \"client/**/*.ts*\"",
     "lint": "prettier --check \"client/**/*.ts*\" && eslint \"client/**/*.ts*\"",
     "prepare": "husky install",
     "preview": "electron-vite dev",
+    "release": "electron-builder",
```

Comment:
@roll great job. one question, though. Why have two scripts with that perform the same action? just for the name? Here `"dist"` and `"release"` are doing the same.

Reply:
That's because of Electron Builder implicit config. On the other hand, we can still make explicit I think

Reply:
https://www.electron.build/publish.html#how-to-publish